src/main/java/seedu/binbash/item/Item.java

```diff
@@ -5,21 +5,23 @@
  * Subclasses can extend Item in order to add properties that are specific to different Item types.
  */
 public abstract class Item {
-    protected final String itemName;
-    protected final String itemDescription;
+    protected String itemName;
+    protected String itemDescription;
     protected int itemQuantity;
-    protected final double itemCostPrice;
+    protected double itemCostPrice;
     protected int totalUnitsPurchased;
+    protected Integer itemThreshold;
```

Comment:
```suggestion
    protected int itemThreshold;
```
and all other occurrences as well.

Reply:
Thanks for noticing, I missed that out as I was refactoring the code.